src/bitcoin/d/mod.rs

```diff
@@ -116,7 +132,11 @@ impl From<simple_http::Error> for BitcoindError {
 }
 
 pub struct BitcoinD {
+    /// Client for generalistic calls.
     node_client: Client,
+    /// A client that will disregard responses to the queries it makes.
+    sendonly_client: Client,
```

Comment:
Ok for the hack, but I feel that we can change it later for a new param for the `make_request` method, instead of allocating a new whole client.

Reply:
Yeah though i checked and allocating a new whole client is just 64 bytes plus this struct:
```rust
pub struct SimpleHttpTransport {
    addr: net::SocketAddr,
    path: String,
    timeout: Duration,
    /// The value of the `Authorization` HTTP header.
    basic_auth: Option<String>,
}
```
So that's definitely reasonable enough in terms of memory footprint.